src/Runner.Worker/ActionManager.cs

```diff
@@ -1,4 +1,4 @@
-using System;
+﻿using System;
```

Comment:
curious, any idea what changed? do you have a BOM in your file?

Reply:
Yep, BOM.  Not sure how that happened, fixed it.